docs/api/v3.rst

```diff
@@ -24,10 +24,22 @@ Token
 The ``Authorization`` HTTP header can be specified with ``Token <your-access-token>``
 to authenticate as a user and have the same permissions that the user itself.
 
-.. note::
 
-   You will find your access Token under
-   `your profile settings <https://readthedocs.org/accounts/tokens/>`_.
+.. tabs::
+
+  .. tab:: Read the Docs Community
```

Comment:
We are using `|org_brand|` and `|com_brand|` when having title like these.

https://github.com/readthedocs/readthedocs.org/blob/master/docs/hosting.rst#L23-L23